core/src/test/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProviderSpec.scala

```diff
@@ -48,6 +49,8 @@ class LocalEntityProviderSpec
     with MockitoTestUtils {
   import driver.api._
 
+  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(3000, Millis)))
+
```

Comment:
I found some tests in this class were timing out intermittently; it was using the default timeout of 150ms. Increasing the timeout made it stable. The changes to streaming likely tweaked timing.